erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md

```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+
+### Prerequisite
+
+You need to create following ledgers in order to post invoice discounting transactions.
+
+**Short Term Loan:** A ledger under the 'Current Liabilities' > 'Loans (Liabilities)' group for loan.
+
+**Bank Account Charges:** An expense ledger for charges levied by the bank.
+
+**Accounts Receivable Credit Account:** A control account of type receivable.
+
+**Accounts Receivable Discounted Account:** A receivable account for invoices which have been discounted.
+
+**Accounts Receivable Unpaid Account:** A receivable account for invoices which were discounted and remain unpaid even after the loan period is over.
+
+
+
+### 1. How to Post an Invoice Discounting Transaction
+
+1. Go to **Accounting > Banking and Payments > Invoice Discounting > New**.
+1. Enter Posting Date and Loan Start Date. Enter the Loan Period in days.
+1. Select invoices either manually in the table or by clicking on the 'Get Invoices' button on the top right.
+1. Select Short Term Loan Account, Bank Account, and Bank Charges Account.
+1. Select Accounts Receivable Credit Account, Accounts Receivable Discounted Account and Accounts Receivable Unpaid Account
+1. Click on Save then Submit.
+1. Click on 'Disburse Loan' and a Journal Entry screen will be prepared. Save then Submit the Journal Entry.
```

Comment:
unclear
After submitting, click on 'Disburse Loan'. Go to Journal Entry....?

also, a screenshot for the journal entry screen?